modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/custom-sql/default.xml

```diff
@@ -100,6 +117,34 @@
 				DDMFormInstance.formInstanceId DESC
 		]]>
 	</sql>
+	<sql id="com.liferay.dynamic.data.mapping.service.persistence.DDMFormInstanceFinder.findByC_G_N_D_P">
+		<![CDATA[
+			SELECT
+				{DDMFormInstance.*}
+			FROM
+				(
+					SELECT
+						DISTINCT DDMFormInstance.formInstanceId
+					FROM
+						DDMFormInstance, DDMFormInstanceVersion
+					WHERE
+						(DDMFormInstance.formInstanceId = DDMFormInstanceVersion.formInstanceId) AND
+						(DDMFormInstance.groupId = ?) AND
+						(DDMFormInstance.companyId = ?) AND
+						(
+							(LOWER(DDMFormInstance.name) LIKE ? [$AND_OR_NULL_CHECK$]) [$AND_OR_CONNECTOR$]
+							(DDMFormInstance.description LIKE ? [$AND_OR_NULL_CHECK$])
+						) AND
+						(DDMFormInstance.settings_ LIKE [$PUBLISHED_FILTER_BEGIN$][$PUBLISHED$][$PUBLISHED_FILTER_END$])
```

Comment:
Can we put all the hardcoded LIKE filter already in the query? We just leave the [$PUBLISHED$] there.

Reply:
Done!